integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -432,6 +446,9 @@ public void updateDatabaseAclPolicies(
         .onErrorResume(
             WebClientResponseException.class,
             e -> Mono.error(new WebClientResponseWithMessageException(e)))
+        .onErrorResume(
+            WebClientRequestException.class,
+            e -> Mono.error(new WebClientRequestWithMessageException(e)))
```

Comment:
Feels like it could be worth factoring out this error handling logic to a single common place -- in particular I am worried that some future API addition may forget to handle one or more of the relevant exceptions, breaking the API contract

Similar to what we have in `handleCreateUpdateHttpError()`